plugins/git/git.plugin.zsh

```diff
@@ -42,6 +42,19 @@ function git_main_branch() {
   echo master
 }
 
+# Check for develop and similar named branches
+function git_develop_branch() {
+  command git rev-parse --git-dir &>/dev/null || return
+  local branch
+  for branch in dev devel development; do
```

Comment:
What if you have all 3 in your repo?

Reply:
https://github.com/ohmyzsh/ohmyzsh/blob/f58e9a55d11477821f69eed57559f945b2773ce9/plugins/git/git.plugin.zsh#L36
The same that would happen if someone had both a `main` and `trunk` branch. The first existing one would get selected.
But I suppose that would be a very rare edge case.